sys/net/gnrc/link_layer/lorawan/include/gnrc_lorawan_internal.h

```diff
@@ -176,6 +176,8 @@ typedef struct {
     uint8_t dev_nonce[2];   /**< Device Nonce */
     uint8_t backoff_state;  /**< state in the backoff state machine */
     uint16_t adr_ack_cnt;   /**< ADR ACK counter */
+    uint8_t adr_req_cnt;     /**< LINK ADR REQ counter */
+    bool adr_req_recv;       /**< Check Link ADR Req was received previously */
```

Comment:
aren't these 2 redundant? it seems `adr_req_recv == (adr_req_cnt > 0)`. Then, please drop `adr_req_recv`

Reply:
Yup. Removed.